runtime/codegen/registry.go

```diff
@@ -62,6 +62,10 @@ type Registration struct {
 	LocalStubFn  func(impl any, tracer trace.Tracer) any
 	ClientStubFn func(stub Stub, caller string) any
 	ServerStubFn func(impl any, load func(key uint64, load float64)) Server
+
+	// String containing the result of MakeEdgeString(Name, Dst) for all
+	// components named Dst used by this component.
+	UseData string
```

Comment:
Might be clearer if renamed to "CallData"

Reply:
A might have weaver.Ref[B] but not call it.  I will rename it to RefData